docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/creating-sensor-description/creating-sensor-description.md

```diff
@@ -0,0 +1,867 @@
+    <robot name="vehicle" xmlns:xacro="http://ros.org/wiki/xacro">
+      <xacro:arg name="config_dir" default="$(find sample_sensor_kit_description)/config"/>
+      <xacro:property name="calibration" value="${xacro.load_yaml('$(arg config_dir)/sensors_calibration.yaml')}"/>
+
+      <!-- sensor kit -->
+      <xacro:include filename="sensor_kit.xacro"/>
+      <xacro:sensor_kit_macro
+        parent="base_link"
+        x="${calibration['base_link']['sensor_kit_base_link']['x']}"
+        y="${calibration['base_link']['sensor_kit_base_link']['y']}"
+        z="${calibration['base_link']['sensor_kit_base_link']['z']}"
+        roll="${calibration['base_link']['sensor_kit_base_link']['roll']}"
+        pitch="${calibration['base_link']['sensor_kit_base_link']['pitch']}"
+        yaw="${calibration['base_link']['sensor_kit_base_link']['yaw']}"
+      />
+      .
+      .
+      .
+    </robot>
+    ```
+
+The tutorial_vehicle file is [sensors.xacro](https://github.com/leo-drive/tutorial_vehicle_sensor_kit/blob/main/tutorial_vehicle_sensor_kit_description/urdf/sensors.xacro) included in [tutorial_vehicle_sensor_kit](https://github.com/leo-drive/tutorial_vehicle_sensor_kit/tree/main) and given below.
+
+??? note "sensors.xacro for tutorial_vehicle_sensor_kit_launch"
+
+    ```yaml
+    <?xml version="1.0"?>
+    <robot name="vehicle" xmlns:xacro="http://ros.org/wiki/xacro">
+      <xacro:arg name="config_dir" default="$(find sample_sensor_kit_description)/config"/>
+      <xacro:property name="calibration" value="${xacro.load_yaml('$(arg config_dir)/sensors_calibration.yaml')}"/>
+
+      <!-- sensor kit -->
+      <xacro:include filename="sensor_kit.xacro"/>
+      <xacro:sensor_kit_macro
+        parent="base_link"
+        x="${calibration['base_link']['sensor_kit_base_link']['x']}"
+        y="${calibration['base_link']['sensor_kit_base_link']['y']}"
+        z="${calibration['base_link']['sensor_kit_base_link']['z']}"
+        roll="${calibration['base_link']['sensor_kit_base_link']['roll']}"
+        pitch="${calibration['base_link']['sensor_kit_base_link']['pitch']}"
+        yaw="${calibration['base_link']['sensor_kit_base_link']['yaw']}"
+      />
+    </robot>
+    ```
+
+## 2. VEHICLE-ID_sensor_kit_launch
+
+VEHICLE-ID_sensor_kit_launch is where the launch files related to sensor startup are stored.
+
+### Folder Structure
+
+??? note "VEHICLE-ID_sensor_kit_launch Folder Structure"
+
+    ```diff
+    VEHICLE-ID_sensor_kit_launch/
+       ├─ config/
+       │     ├─ diagnostic_aggregator/
+       │     │     └─ sensor_kit.param.yaml
+       │     └─ dummy_diag_publisher/
+       │           └─ sensor_kit.param.yaml
+       ├─ data/
+       │     └─ traffic_light_camera.yaml
+       └─ launch/
+             ├─ camera.launch.xml
+             ├─ gnss.launch.xml
+             ├─ imu.launch.xml
+             ├─ lidar.launch.xml
+             ├─ pointcloud_preprocessor.launch.py
+             └─ sensing.launch.xml
+    ```
```

Comment:
There is no need for `note` element. Also you can add "+" tag to created wile for `diff`.
```suggestion

    ```diff
    VEHICLE-ID_sensor_kit_launch/
          ├─ config/
          │     ├─ diagnostic_aggregator/
     +   │     │     └─ sensor_kit.param.yaml
          │     └─ dummy_diag_publisher/
     +   │           └─ sensor_kit.param.yaml
          ├─ data/
     +   │     └─ traffic_light_camera.yaml
          └─ launch/
     +        ├─ camera.launch.xml
     +        ├─ gnss.launch.xml
     +        ├─ imu.launch.xml
     +        ├─ lidar.launch.xml
     +        ├─ pointcloud_preprocessor.launch.py
     +        └─ sensing.launch.xml
    ```
```